### ISSUE TITLE
excludeRoutes has no impact on index

### ISSUE THREAD

**Issue by erzz** [praveenn77/docusaurus-lunr-search#133]
```
const config = {
 // stuff
  plugins [
   [
      'docusaurus-lunr-search',
      {
        languages: ['en'],
        highlightResults: true,
        maxHits: 8,
        excludeRoutes: [
          'docs/e2e-examples/**/*', // prefer workflows over e2e-examples
          'docs/v1/**/*', // prefer v2 over v1
        ],
      },
    ],
// stuff
}
```

Yet I still have results from those paths in my index and no matter what I set the paths to (even **/*) I see `indexed 106 documents out of 157`

**Comment by klauss194** [comment]
I have the same issue -> not sure how to fix it

**Comment by praveenn77** [comment]
new version published with fix `2.3.2`

**Comment by erzz** [comment]
Much appreciated! Working perfect here!
